emrun.py

```diff
@@ -424,6 +432,9 @@ def kill_browser_process():
 # starting a browser process from command line generally launches just a "stub" spawner
 # process that immediately exits.
 def detect_browser_processes():
+  if not browser_exe:
+    return # Running with --no_browser, we are not binding to a spawned browser.
```

Comment:
Can we land this specific fix separably?   This seems like that fix for https://github.com/emscripten-core/emsdk/issues/712 which is unrelated to running under python 2 or out-of-tree builds, right?

Reply:
Or we could just land this PR? I don't see there being too much value to split PRs to atoms.

Reply:
It just that it seems completely unrelated to the rest of the stuff in this PR (and the PR title).    I guess I don't mind mixing them up but this part seems so completely trivial and easy to land right now,  whereas the other stuff here related to making emsdk standalone and python2-compatible we are having ongoing discussions about.